src/ui/views/LogsView/logEntryContext.tsx

```diff
@@ -0,0 +1,25 @@
+import { Box, Typography } from '@mui/material';
+import React, { FunctionComponent } from 'react';
+
+const LogEntryContextComponent: FunctionComponent<object> = (context) => {
```

Comment:
Do not spread context object to react component. Just pass it as a single argument.